frontend/src/metabase/entities/containers/EntityName.unit.spec.tsx

```diff
@@ -1,32 +1,32 @@
 import {
   setupCardEndpoints,
-  setupUserEndpoints,
+  setupDashboardEndpoints,
 } from "__support__/server-mocks";
 import { createMockEntitiesState } from "__support__/store";
 import { renderWithProviders, screen } from "__support__/ui";
-import { createMockCard, createMockUser } from "metabase-types/api/mocks";
+import { createMockCard, createMockDashboard } from "metabase-types/api/mocks";
 
 import { EntityName } from "./EntityName";
 
 describe("EntityName", () => {
-  describe("users", () => {
-    test("user with name", async () => {
-      const mockUser = createMockUser({
-        common_name: "Testy Tableton",
+  describe("dashboards", () => {
```

Comment:
Did this change because this test requires something from the entity framework?

Reply:
Yeah `EntityName` needed the `User` entity to exist. Rather than delete the test I just made it work for another entity we use this component for.